db/functions/redundant.py

```diff
@@ -4,18 +4,33 @@
 Mathesar filters not supporting composition.
 """
 
-from db.functions.base import DBFunction
+from abc import abstractmethod
+
+from db.functions.base import DBFunction, Or, Lesser, Equal, Greater, StartsWith, Contains, ToLowercase
 
 from db.functions import hints
 
 
-# TODO (tech debt) define in terms of other DBFunctions
-# would involve creating an alternative to to_sa_expression: something like to_db_function
-# execution engine would see that to_sa_expression is not implemented, and it would look for
-# to_db_function.
+class RedundantDBFunction(DBFunction):
```

Comment:
I find this name confusing. Perhaps something like `DBFunctionCombination` would be clearer?

Reply:
I think DBFunctionCombination can be even more confusing, since a regular DBFunction instance can contain other DBFunction instances as parameters, and thus could be called a combination or supporting combination. I like redundant, because it highlights that it's made of DBFunctions that could be used without this particular DBFunction. I'm open to other names. Maybe `SecondaryDBFunction`?

Reply:
I like `DBFunctionPackage`.